packages/components/htz-components/src/components/TeaserResponsiveText/TeaserResponsiveText.js

```diff
@@ -18,32 +18,35 @@ TeaserResponsiveText.defaultProps = {
   displayStrict: false,
 };
 
-const mobileStyle = ({ theme, }) => ({
-  ...theme.mq({ from: 's', }, { display: 'none', }),
-});
-const aboveMobileStyle = ({ theme, }) => ({
-  ...theme.mq({ until: 's', }, { display: 'none', }),
-});
-
 export default function TeaserResponsiveText({
   text,
   mobileText,
   displayStrict,
 }: TextPropTypes): React.Node {
-  return (mobileText || text) && text !== mobileText ? (
+  const { css, theme, } = useFela();
+
+  // if there is no text or mobile text return nothing
+  if (!(mobileText || text)) return null;
+  // if there is text but no mobile text, and we are not in strict mode, return text for all breakpoints
+  // else, continue on
+  if (text && !mobileText && !displayStrict) {
+    return <React.Fragment>{text}</React.Fragment>;
```

Comment:
Why the `<Fragment>` here? Any reason to not just return`text`?
```suggestion
    return text;
```

Reply:
**After refactor**
When no React.Fragment. Flow error: 
Cannot return `text || mobileText` because:
- Either null or undefined [1] is incompatible with null [2].
- Or property `@@iterator` is missing in null or undefined [1] but exists in `$Iterable` [3]. 

(Iterable<?React$Node>)